Providers/Profile.swift

```diff
@@ -137,17 +125,12 @@ protocol Profile: AnyObject {
 
     func removeAccount()
 
-    func getClients() -> Deferred<Maybe<[RemoteClient]>>
-    func getCachedClients()-> Deferred<Maybe<[RemoteClient]>>
     func getClientsAndTabs() -> Deferred<Maybe<[ClientAndTabs]>>
-
     func getCachedClientsAndTabs(completion: @escaping ([ClientAndTabs]) -> Void)
     func getCachedClientsAndTabs() -> Deferred<Maybe<[ClientAndTabs]>>
 
     func cleanupHistoryIfNeeded()
 
-    func sendQueuedSyncEvents()
```

Comment:
Same here, do you not want this to be part of protocol as I see its still part of BrowserProfile class

<img width="796" alt="Screen Shot 2022-09-02 at 4 04 41 PM" src="https://user-images.githubusercontent.com/8919439/188228239-29f7c5d5-cb86-4243-9d17-fe4010fa11bd.png">

Reply:
Same as above.